File: mobile/src/main/java/net/mm2d/dmsexplorer/domain/tabs/KeepAliveService.java
/*
 * Copyright (c) 2018 大前良介 (OHMAE Ryosuke)
 *
 * This software is released under the MIT License.
 * http://opensource.org/licenses/MIT
 */

package net.mm2d.dmsexplorer.domain.tabs;

import android.app.Service;
import android.content.Intent;
import android.os.Binder;
import android.os.IBinder;
import android.support.annotation.Nullable;

/**
 * @author [大前良介 (OHMAE Ryosuke)](mailto:ryo@mm2d.net)
 */
public class KeepAliveService extends Service {
    private static final Binder sBinder = new Binder();

    @Nullable
    @Override
    public IBinder onBind(final Intent intent) {
        return sBinder;
    }
}

File: mobile/src/main/java/net/mm2d/dmsexplorer/domain/tabs/KeepAliveService.kt
/*
 * Copyright (c) 2018 大前良介 (OHMAE Ryosuke)
 *
 * This software is released under the MIT License.
 * http://opensource.org/licenses/MIT
 */

package net.mm2d.dmsexplorer.domain.tabs

import android.app.Service
import android.content.Intent
import android.os.Binder
import android.os.IBinder

/**
 * @author [大前良介 (OHMAE Ryosuke)](mailto:ryo@mm2d.net)
 */
class KeepAliveService : Service() {
    override fun onBind(intent: Intent): IBinder? {
        return sBinder
    }

    companion object {
        private val sBinder = Binder()
    }
}
